src/Sfa.Tl.Matching.Functions/local.settings.json

```diff
@@ -3,6 +3,7 @@
     "Values": {
         "FUNCTIONS_WORKER_RUNTIME": "dotnet",
         "CronSchedule": "0 */15 * * * *",
+        "BankHolidayGemeratorTrigger": "0 0 9 * * MON-FRI",
```

Comment:
Type in Generator

Reply:
I thought I fixed all of those! It's done now.